src/tables.js

```diff
@@ -0,0 +1,91 @@
+const indexOf = Array.prototype.indexOf
+const rules = {}
+
+rules.tableCell = {
+  filter: ['th', 'td'],
+  replacement: function (content, node) {
+    return cell(content, node)
+  }
+}
+
+rules.tableRow = {
+  filter: 'tr',
+  replacement: function (content, node) {
+    let borderCells = ''
+    const alignMap = { left: ':--', right: '--:', center: ':-:' }
+
+    if (isHeadingRow(node)) {
+      for (let i = 0; i < node.childNodes.length; i++) {
+        let border = '---'
+        const align = (
+          node.childNodes[i].getAttribute('align') || ''
+        ).toLowerCase()
+
+        if (align) border = alignMap[align] || border
+
+        borderCells += cell(border, node.childNodes[i])
+      }
+    }
+    return '\n' + content + (borderCells ? '\n' + borderCells : '')
+  }
+}
+
+rules.table = {
+  // Only convert tables with a heading row.
+  // Tables with no heading row are kept using `keep` (see below).
+  filter: function (node) {
+    return node.nodeName === 'TABLE' && isHeadingRow(node.rows[0])
```

Comment:
I'm not sure that I really understand this comment

Reply:
That's because it's no longer relevant, I suspect

Reply:
I've removed the comment